changelog.d/365.misc

```diff
@@ -0,0 +1 @@
+Convert sydent/http/srvresolver.py and associated files to async/await.
```

Comment:
It would make sense to use more generic changelogs for some of these as towncrier (the tool we use during releases to make the changelog) combines them automatically if they're identical.

#364 used:

> Convert `inlineCallbacks` to async/await.

So if we use the same thing here the changelog entry will automatically end up as:

> * Convert `inlineCallbacks` to async/await. (#364, #365)